spirv_glsl.cpp

```diff
@@ -3258,6 +3301,292 @@ void CompilerGLSL::emit_resources()
 	declare_undefined_values();
 }
 
+void CompilerGLSL::emit_extension_workarounds()
+{
+	const auto TYPES = { "int",   "ivec2", "ivec3", "ivec4", "uint",   "uvec2", "uvec3", "uvec4",
```

Comment:
It can be awkward to deduce what type this is. I suggest `static const char *[]` here.